src/prototype/WingetCreateMakeDSC.ps1

```diff
@@ -0,0 +1,63 @@
+# This script is a prototype for quickly creating DSC files.
+
+if ($null -eq (Get-InstalledModule -Name Microsoft.Winget.Client))
+{
+  Install-Module Microsoft.Winget.Client
```

Comment:
Does this still require PS Core (v7)? If so, should probably add a #Requires statement to ensure the script can only run in Powershell 7